jobserver/migrations/0013_remove_interactivereporter.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 5.1.6 on 2025-03-19 12:00
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+
+from jobserver.authorization.roles import InteractiveReporter
+from jobserver.commands import project_members
+from jobserver.models import ProjectMembership
+
+
+def remove_interactive_reporter_project_role(apps):
+    # The ProjectMembership model has the ImmutableModelMixin
+    # which provides an `override` parameter to the overridden model save()
+    # (which is absent in the base model save() method)
+    # and is provided when called within update_roles().
+    # The recommended way of accessing django model classes in migrations
+    # is using apps.get_model(), however this just returns a vanilla model
+    # class with no mixins and so the call to save(override=True) fails.
```

Comment:
I'm not sure that I understand this comment. The call to `project_members.update_roles` appears to work as expected, but you state the call to `save` failed?

Reply:
Update roles calls save within itself, but it expects the save that's overridden in that mixin not the base save